src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -4604,6 +4604,26 @@ class C
             End Using
         End Function
 
+        <MemberData(NameOf(AllCompletionImplementations))>
+        <WpfTheory, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Async Function TestVariableNameInCurlyBraces(completionImplementation As CompletionImplementation) As Task
+            Using state = TestStateFactory.CreateCSharpTestState(completionImplementation,
+                              <Document>
+class C
+{
+    public static void M() 
+    {
+        var a = 1;
+        if (a is { $$})
+    }
+}
+                              </Document>)
+
+                state.SendTypeChars("a")
+                Await state.AssertSelectedCompletionItem(displayText:="a")
```

Comment:
wait... why?  Why would 'a' be appropriate here?  Isn't this a recursive property pattern?  in which case, i would only expect the members of 'int' to be shown here.  

I don't believe this is a general expression context.

Reply:
* Yes, currently, only members of `int `are displayed. 
* In 15.9, it was an expression context and displayed `a` and tons of other symbols.
* I understand that the sample in the unit test maybe not an example we need to cover. However, it changed the same way as the one we need:
1. Set a break point
2. Right click on it
3. Choose `Actions`
4. In 'Log a message to Output Window', type "`The value of a is { $$`"
5. Replace $$ with the caret and type `a`

**Expected**
Completion contains `a`

**Actual**
No completion provided

Do you mean we should reduce the scope of the issue to the breakpoint scenario?

Reply:
> In 15.9, it was an expression context and displayed a and tons of other symbols.

That seems... bad

> Yes, currently, only members of int are displayed.

That seems.... good :)

> I understand that the sample in the unit test maybe not an example we need to cover. However, it changed the same way as the one we need:

But... why do we want this behavior?

Reply:
> In 'Log a message to Output Window', type `The value of a is { $$`

I don't understand what's happening there.  That looks like free-form text.  It's not C#.  What would happen if they wrote: `the value of a has become { $$`?  there's no `is` there, so you won't have a property pattern.

Reply:
> But... why do we want this behavior?

We need this behavior in Edit BreakPoint -> Actions -> Log a message to Output Window. I thought that if it changed in all scopes, we may need to fix it in all scopes.

>> In 15.9, it was an expression context and displayed a and tons of other symbols.
> That seems... bad
>
> > Yes, currently, only members of int are displayed.
> That seems.... good :)

I am also not sure that `is {` guarantees the it is followed by a valid property pattern.
Currently, in 16.0, there are two items in the completion popup: `MinValue `and `MaxValue`. Using them leads to the following syntax:
`if (a is { MinValue: 5, MaxValue: 13 })`
which is invalid, as far as I see even in C# 8.0. 

So, why should we restrict the completion to `MinValue `and `MaxValue` only in the case?